hwdb.d/60-sensor.hwdb

```diff
@@ -116,9 +116,10 @@ sensor:modalias:acpi:BOSC0200*:dmi:*svnAcer*:*pnSpinSP111-34:*
 # Aquarius
 #########################################
 
-# Aquarius Cmp NS483
```

Comment:
Why are these comments being removed? It is useful to note the models

Reply:
The Cmp NS483 and NS483 models differ, so in the last pull request I divided all the sensors into sections, but apparently I misunderstood you, I added the minimum necessary in these changes, and deleted the comments of the models. it seemed to me that in this case they would be confusing.. I understood what was needed, I'll fix it now.